packages/core/contracts/cross-chain-oracle/GovernorSpoke.sol

```diff
@@ -1,20 +1,21 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 pragma solidity ^0.8.0;
 
-import "./ChildMessengerInterface.sol";
+import "./interfaces/ChildMessengerConsumerInterface.sol";
 import "../common/implementation/Lockable.sol";
 
 /**
- * @notice Governor contract deployed on sidechain that receives governance actions from Ethereum.
+ * @title Cross-chain Oracle L2 Governor Spoke.
+ * @notice Governor contract deployed on L2 that receives governance actions from Ethereum.
  */
-contract GovernorSpoke is Lockable {
+contract GovernorSpoke is Lockable, ChildMessengerConsumerInterface {
```

Comment:
This PR introduces two new interfaces `ChildMessengerConsumerInterface` and `ParrentMessengerConsumerInterface`. These go in the L2 an L1 contracts, respectively, where the contracts "consume" the data from the messenger. For example, in this `GovernorSpoke` we are "consuming" data from the messenger in the `processMessageFromParent` function. This lets us `override` this function here. this is required within the `messenger` contracts to let us define the interface that is called within the consumer contract.